src/client/debugger/jupyter/debuggingManager.ts

```diff
@@ -345,22 +395,31 @@ export class DebuggingManager implements IExtensionSingleActivationService, IDeb
                 controller?.connection.interpreter
             );
 
+            if (result === ProductInstallStatus.Installed) {
+                sendTelemetryEvent(DebuggingTelemetry.ipykernel6Status, undefined, { status: 'installed' });
+            } else {
+                sendTelemetryEvent(DebuggingTelemetry.ipykernel6Status, undefined, { status: 'notInstalled' });
+            }
             return result === ProductInstallStatus.Installed;
         } catch {
             return false;
         }
     }
 
-    private async installIpykernel6(doc: NotebookDocument) {
+    private async installIpykernel6() {
         const response = await this.appShell.showInformationMessage(
             DataScience.needIpykernel6(),
             { modal: true },
-            DataScience.jupyterInstall()
+            DataScience.setup()
         );
 
-        if (response === DataScience.jupyterInstall()) {
-            const controller = this.notebookControllerManager.getSelectedNotebookController(doc);
-            void this.installer.install(Product.ipykernel, controller?.connection.interpreter, undefined, true);
+        if (response === DataScience.setup()) {
+            sendTelemetryEvent(DebuggingTelemetry.clickedOnSetup);
+            this.appShell.openUrl(
```

Comment:
This seems crappy to me. You try to debug and now you have to go to a website to try and install ipykernel 6? We can't do a best effort and then if it fails send them to the website?

Reply:
I had this conversation with Claudia. Some people might quit if the installer doesn't work. That's double the annoyance, try to debug, click 'Intall', fail, have to go to a website. We think this way is more effective until we can fix the installer in the python extension.

Reply:
Ok, sounds like it was discussed, but I have to say, seems pretty crummy for a user experience. Did the installer really fail that often? Was this just for the older users on 3.6?

Reply:
Sometimes it installs it in the wrong conda env.
Sometimes it fails to activate the conda env because its using powershell.
If the python version is 3.6 or lower, the installation is a 'success' but you didn't actually upgrade to ipykernel 6, but to 5.5.5.

There might be more, but that's what I've seen so far.

Reply:
I know its not ideal, its giving some trust to the user that they know how to navigate python envs. The bet is that they'll do it, after all, they're using notebooks and want to debug them.

Reply:
Do we have telemetry on number of times ipykernel 6 install was tried and corresponding success on debugging a cell?

Reply:
We don't, but we will starting next month. This PR has new telemetry for that.